### PR TITLE
feat(INJI-419): notify user on minStorageLimitReached while vc download

### PR DESCRIPTION
**Pull request by KiruthikaJeyashankar**
When user clicks on plus icon in home page for downloading VC, if the device storage is less than the min threshold defined in our inji-default.properties file, error message overlay is shown (attached screenshot)

mosip.inji.minStorageRequired=5

image

### CODE REVIEW

**Review by swatigoel — APPROVED** [review]

File: screens/Home/HomeScreen.tsx [hunk 1]
```diff
@@ -76,6 +77,12 @@ export const HomeScreen: React.FC<HomeRouteProps> = props => {
         )}
       </Column>
       {isOpenId4VCIEnabled() && <DownloadFABIcon />}
+      <ErrorMessageOverlay
```
Comment (vijay151096) [inline screens/Home/HomeScreen.tsx:80]: Have we removed the overlay from MyVcsTab.tsx, which was used earlier in the downloadcard scenario

Comment (KiruthikaJeyashankar) [reply screens/Home/HomeScreen.tsx:80]: We have not cleaned up the download card button still so I have not removed this storage error overlay as well from MyVCsTab.tsx

Comment (vijay151096) [reply screens/Home/HomeScreen.tsx:80]: okay, can you please add this as a sub task in the same clean up story

Comment (KiruthikaJeyashankar) [reply screens/Home/HomeScreen.tsx:80]: Noted

**Pull request merged**